app/controllers/endorsements_controller.rb

```diff
@@ -14,7 +15,9 @@ def create
     end
 
     user_skill = UserSkill.find_by(id: endorsement_params[:userSkillId])
-    render json: {skill: user_skill.skill.name, count: user_skill.endorsements.count, endorseButton: endorse_button}
+    render json: {skill: user_skill.skill.name,
+      count: user_skill.endorsements.count,
+      endorseButton: endorse_button}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.